Makefile

```diff
@@ -0,0 +1,16 @@
+DOCKER := docker run -it --rm quixo-python
+
+help: ## Print all commands (default)
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+install: ## Build the docker container
+	docker build -t quixo-python .
+
+run: ## Run the game. Usage : FILE_STATE=your-filename make run []
+	$(DOCKER) python3 ./main.py $(FILE_STATE)
```

Comment:
I'm not sure the FILE_STATE is required here.
I guess that you have copy/pasted it this somewhere, but you don't need it.